.cirrus.yml

```diff
@@ -76,6 +76,37 @@ base_template: &BASE_TEMPLATE
     - git checkout FETCH_HEAD  # Use merged changes to detect silent merge conflicts
                                # Also, the merge commit is used to lint COMMIT_RANGE="HEAD~..HEAD"
 
+worker_template: &WORKER_TEMPLATE
+  persistent_worker:
+    isolation:
+      vetu:
+        image: ghcr.io/cirruslabs/ubuntu-runner-amd64:22.04
+        user: admin
+        password: admin
+        disk_size: 50
+
+small_worker_template: &SMALL_WORKER_TEMPLATE
+  << : *WORKER_TEMPLATE
+  persistent_worker:
+    isolation:
+      vetu:
+        cpu: 2
+        memory: 8G
```

Comment:
What if someone sets memory in a malicious PR to 63 GB (out of our 64GB) and keeps 8 GB of memory in the resources field? Would we allocate 63GB of memory to one VM and the runner would OOM? Crashing the server and/or the cirrus runner (and thus other jobs)?

Reply:
I think, for these, we rely on Github's "Require admin approval for first-time contributors" security setting (and our fellow contributors not going rogue). Can't think of a much better defence, other than using one runner per worker

Reply:
although, don't know if this works for cirrus...